build.gradle.kts

```diff
@@ -25,10 +25,9 @@ kotlin {
 
 dependencies {
     intellijPlatform {
-        webstorm("2023.3")
-        bundledPlugins("JavaScript")
+        webstorm("2025.1")
```

Comment:
previously you asked if I was sure it was working in webstorm 2023.3 - yeah it was in the previous PR but I didn't check if I was getting an error message in the plugins page. The updated PR is for 2025 as that's no longer in preview.
Updated to 2025.1, I don't think it will be backwards compatible with 2023 now.